lib/estimator/estimator.rb

```diff
@@ -0,0 +1,89 @@
+module Estimator
+  class Estimate < ActiveRecord::Base
+    require 'active_shipping'
+
+    def self.query(ship_params)
+      @api_call_ok = true
+      ship_array = shipment_array(ship_params)
+      ups_rates = get_rates(ship_array, ups)
+      usps_rates = get_rates(ship_array, usps)
+      quote = assemble_quote(ups_rates, usps_rates)
+      if @api_call_ok
+        return quote
+      else
+        return nil
+      end
+      @api_call_ok = true
```

Comment:
Please stick me if I missed something, but this seems a little redundant. Since you don't change the value of instance variable anywhere in the method, you shouldn't need to reassign it's value to true again.